### PR TITLE
release/v0.6.1

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR includes changes raised within the following accompanying PRs:
- PR #38 
- PR #37 

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide an explanation as to how the change is non-breaking below.)

Please see relevant PRs linked for more details

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

Please see relevant PRs linked for more details

**How did you test the PR changes?** 

 

- [X] Buildkite  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🔗 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -88,6 +88,3 @@ seeds:
     team_data:
       +column_types:
         id: "{{ 'int64' if target.type == 'bigquery' else 'bigint' }}"
-clean-targets:
-- target
-- dbt_modules
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:93]: Removed these add the clean target has changed to dbt_packages and that is used by default. Therefore there is no need to include them anymore. It also threw a weird warning when regenerating the docs.

File: packages.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 packages:
-- package: fivetran/intercom_source
-  version: [">=0.6.0", "<0.7.0"]
-
+# - package: fivetran/intercom_source
+#   version: [">=0.6.0", "<0.7.0"]
+  - git: https://github.com/fivetran/dbt_intercom_source.git
+    revision: release/v0.6.1
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: Obligatory note to update before merge.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+# dbt_intercom v0.6.1
+
+## 🐛 Bug Fixes 🐛
+- Adjusted the `first_admin_response_at` and `last_admin_response_at` logic within `int_intercom__conversation_part_aggregates` to take into account that admin replies may sometimes exist within assignment parts. This is differentiated from normal assignments by also checking to determine if the body of the part is populated. If it is, then it is safe to assume this is a true admin reply and should be counted as such.
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:4]: Sorry I didn't catch this in the initial PR. 

```suggestion
- Adjusted the `first_admin_response_at` and `last_admin_response_at` logic within `int_intercom__conversation_part_aggregates` to take into account that admin replies may sometimes exist within assignment parts. This is differentiated from normal assignments by also checking to determine if the body of the part is populated. If it is, then it is safe to assume this is a true admin reply and should be counted as such. ([#38](https://github.com/fivetran/dbt_intercom/pull/38))
```

**Review by fivetran-catfritz — APPROVED** [review]
One small changelog change. Ran and test locally. Reviewed merge. Besides changelog update, LGTM!

File: packages.yml (hunk 2 above)
Comment (fivetran-catfritz) [inline packages.yml:6]:
```suggestion
 - package: fivetran/intercom_source
   version: [">=0.6.0", "<0.7.0"]
```

**Pull request merged**